Casks/p/positron.rb

```diff
@@ -4,7 +4,7 @@
 
   url "https://github.com/posit-dev/positron/releases/download/#{version}/Positron-#{version}.dmg"
   name "Positron"
-  desc "VScode-based data science IDE"
+  desc "Next-generation data science IDE"
```

Comment:
Thanks @tdstein - we avoid "marketing" terms in descriptions here.
Perhaps we should use;

```suggestion
  desc "Data science IDE"
``

Reply:
That works for me @bevanjkay. Thanks for taking a look.